css-three-column-layout/styles.css

```diff
@@ -0,0 +1,46 @@
+.contaier {
+  display: flex;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+h2 {
+  font-size: 36px;
+}
+
+body {
+  background-color: #3894dc;
+  font-family: 'Lato', sans-serif;
+  color: white;
+  text-align: center;
+}
+
+.full-column {
+  margin: 1rem;
+}
+
+.image {
+  width: 100%;
+  height: 100%;
+}
+
+.one-third-column {
+  margin: 1rem;
+  padding: 1rem;
+}
+
+.circle {
+  width: 100px;
+  height: 100px;
+  background: white;
+  border-radius: 50%;
+  padding: 1.5rem;
+  margin: auto;
+
+  /* margin: 1rem; */
```

Comment:
Please delete any commented code prior to submitting for review

Reply:
Will do 
![Screen Shot 2021-06-28 at 12 28 50 PM](https://user-images.githubusercontent.com/52221756/123693066-6f184200-d80c-11eb-8925-094b0b33a2cd.png)